src/lib/api-endpoints.ts

```diff
@@ -49,7 +49,7 @@ export function searchResourcesEndpoint(
         `/resources/search?languageId=${languageId}&query=${query}${
             resourceTypesParams ? '&' + resourceTypesParams : ''
         }`,
-        1,
```

Comment:
Added this too, forgot to cache-bust this endpoint since the response format changed.